### DISCUSSION TITLE
How can I activate Lua Editor?

### DISCUSSION THREAD

**Discussion by itachi-73m** [o3de/o3de#3466] [General]
hi,Leva Editor is not active for me. If you know the way, please hellp
![Capture](https://user-images.githubusercontent.com/70795789/130845565-2c69ba83-fd3d-417d-aecc-8aff2e8108eb.PNG)

**Comment by lsemp3d** [comment]
There is a problem with the Lua editor that is causing to fail to launch, the issue is being investigated: https://github.com/o3de/o3de/issues/3681
